tests/foreman/ui_airgun/test_contenthost.py

```diff
@@ -60,6 +60,10 @@
     SatelliteToolsRepository,
 )
 from robottelo.vm import VirtualMachine
+from robottelo.api.utils import (
+    wait_for_tasks,
+    )
```

Comment:
you could make this a one line import

Reply:
@JacobCallahan Thank you for that review. Change made and git commits squashed.